volpiano_display_utilities/latin_word_syllabification.py

```diff
@@ -235,7 +237,7 @@ def _get_syl_bound_position(ltrs_btw_vow_grps: str) -> Tuple[int, str]:
     elif num_ltrs_btw_vow_grps == 1:
         split_case = "1 consonant between vowels"
     elif num_ltrs_btw_vow_grps == 2:
-        if ltrs_btw_vow_grps not in _CONSONANT_GROUPS:
+        if ltrs_btw_vow_grps not in _CONSONANT_GROUPS or ltrs_btw_vow_grps == "nc":
```

Comment:
It took me writing out this comment to convince myself that this is equivalent to `if (ltrs_btw_vow_grps not in _CONSONANT_GROUPS) or (ltrs_btw_vow_grps == "nc")` and there's no ambiguity. Could you add parens to this condition, just for clarity?